sktime/forecasting/compose/_pipeline.py

```diff
@@ -236,11 +236,39 @@ class ForecastingPipeline(_Pipeline):
     to X. The forecaster can also be a TransformedTargetForecaster containing
     transformers to transform y.
 
+    For a list `t1`, `t2`, ..., `tN`, `f`, `tp1`, `tp2`, ..., `tpM`
```

Comment:
1. you mean "list of transformers in `steps`
2. The `ForecastingPipeline` does not allow post-processing transformers.